teaspoons-cli/teaspoons/commands/pipelines_commands.py

```diff
@@ -0,0 +1,30 @@
+# commands/pipelines_commands.py
+
+import click
+
+from logic import pipelines_logic
+from utils import _pretty_print, handle_api_exceptions
+
+
+@click.group()
+def pipelines():
+    """Commands for running Teaspooons pipelines"""
+
+
+@handle_api_exceptions
+@pipelines.command()
+def list():
+    """List all available pipelines"""
+    pipelines_list = pipelines_logic.list_pipelines()
+    click.echo(f"Found {len(pipelines_list)} available pipeline{'' if len(pipelines_list) == 1 else 's'}:")
+    for pipeline in pipelines_list:
+        click.echo(f"\t{pipeline.pipeline_name} - {pipeline.description}")
+
+
+@handle_api_exceptions
```

Comment:
A small thing, but might be useful in debugging: decorators in python are applied from bottom to top, so may be it'd be better to decorate with `handle_api_exception` first and then with click-related stuff. Or even better to transfer `handle_api_exception` decoration to `pipelines_logic.list_pipelines()`, this would make this function only responsible for cli stuff.

Reply:
that's a good thought, thanks. we did originally have `handle_api_exception` on the inner (logic) function, but we moved it to this one because it deals more with how to handle/return the error to the user, which does feel like it's "cli stuff," and we didn't want the logic functions to have any knowledge of click (which we'd use to return the error messages). do you feel strongly that the error handling belongs on the logic function?

Reply:
Honestly, I'm not entirely certain. I wouldn’t strongly insist on it either way. Maybe the best way to think about this is to ask: if you were to use the `list_pipelines` function outside of the CLI, would you want this handling logic included? Just something to consider!

Also, one more thing if I am not mistaken, your current `handle_api_exceptions` doesn't use click, it just prints to the user through python's `print`.